src/styles/icons/ab_icons.css

```diff
@@ -315,9 +315,6 @@
 .icon-ab-sidebar-campaigns-2:before {
   content: "\e95f";
 }
-.icon-ab-config-promoter:before {
-  content: "\e960";
-}
 .icon-ab-arrange-top:before {
```

Comment:
please recheck accelicons.svg, arrange-top and arrange-bottom content should change